10/jdk/alpine/Dockerfile.hotspot.releases.slim

```diff
@@ -71,10 +68,14 @@ RUN set -eux; \
     mv ${jdir}/* /opt/java/openjdk; \
     export PATH="/opt/java/openjdk/bin:$PATH"; \
     apk --update add --no-cache bash binutils; \
-    /usr/local/bin/slim-java.sh /opt/java/openjdk; \
-    apk del bash binutils; \
-    rm -rf /var/cache/apk/*; \
-    rm -rf ${jdir} /tmp/openjdk.tar.gz;
+    /usr/local/bin/slim-java.sh /opt/java/openjdk;
+
+FROM alpine:3.7
+
+MAINTAINER Dinakar Guniguntala <dinakar.g@in.ibm.com> (@dinogun)
+
+COPY --from=prepare /opt/java/openjdk /opt/java/openjdk
```

Comment:
to what `prepare` refers to ? Previous build is not named, this is some default ?

Reply:
yes I forgot to name the first stage `prepare`

Reply:
@cesartl you just copying here `openjdk` files, what with dependencies ? `glibc` and other are needed in runtime